engines/myst3/myst3.cpp

```diff
@@ -1479,6 +1483,10 @@ void Myst3Engine::dragItem(uint16 statusVar, uint16 movie, uint16 frame, uint16
 	}
 }
 
+bool Myst3Engine::canSaveGameStateCurrently() {
+	return canLoadGameStateCurrently() && !(_state->getVar(62) == 901 && _state->getVar(67)==0);
```

Comment:
Please use the named versions of the variable accessors for better readability (getLocationRoom and getMenuSavedAge).